.changeset/late-wasps-nail.md

```diff
@@ -0,0 +1,17 @@
+---
+'@xstate/vue': minor
+---
+
+Added new `useActor`, which is a composable that subscribes to emitted changes from an existing `actor`:
+
+```js
+import { useActor } from '@xstate/vue';
+
+export default defineComponent({
+  props: ['someSpawnedActor'],
```

Comment:
is this correct? how would this work? 🤔

Reply:
It's the most basic way to pass a prop to a component! Not recommended with Typescript but was to simplify and focus on the hook itself :)
https://v3.vuejs.org/guide/component-props.html#prop-types

But could change it to a more detailed prop!

Reply:
It's OK then - quick followup question: is this *required*? or does it only enable some static/runtime checks? I guess this might be required by the compiler, but I'm wondering if that's the case.

Reply:
If you want to use the prop in the component it is required. You can choose to do it in an array format or object.

Reply:
It's the same idea as a prop in React that you have to put in the function argument ;)